src/db/QDjangoQuerySet.cpp

```diff
@@ -343,6 +343,17 @@ bool QDjangoQuerySetPrivate::sqlLoad(QObject *model, int index)
  */
 QDjangoQuery QDjangoQuerySetPrivate::countQuery() const
```

Comment:
if countQuery becomes this one-liner, I am not convinced we need to keep the method. It can probably be inlined into ::count()